scripts/schemas/twister/platform-schema.yaml

```diff
@@ -24,7 +24,8 @@ mapping:
     enum: ["mcu", "qemu", "sim", "unit", "native"]
   "simulation":
     type: str
-    enum: ["qemu", "simics", "xt-sim", "renode", "nsim", "mdb-nsim", "tsim", "armfvp", "native"]
+    enum: ["qemu", "simics", "xt-sim", "renode", "nsim", "mdb-nsim", "tsim", "armfvp", "native",
+           "robot"]
```

Comment:
why is this needed?

Reply:
Thanks for spotting it, this was a leftover from the initial implementation.